utils/terrascript_client.py

```diff
@@ -1606,15 +1620,17 @@ def populate_tf_resource_cloudwatch(self, resource, namespace_info):
             tf_resources.append(data('aws_elasticsearch_domain',
                                      es_identifier, **es_domain))
 
-            zip_url = common_values.get('zip_url', LOGTOES_ZIP)
-            r = requests.get(zip_url)
-            open('/tmp/LogsToElasticsearch.zip', 'wb').write(r.content)
+            release_url = common_values.get('release_url', LOGTOES_RELEASE)
```

Comment:
i like this! you are allowing users to override the default value. nice!

Reply:
The default value will be `repos/app-sre/logs-to-elasticsearch-lambda/releases/latest`, while the user could set to `repos/app-sre/logs-to-elasticsearch-lambda/releases/tags/1.0.2` to pin source code to a certain tag, just as we pin saas file to a certain hash.